chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertificationCriterion.java

```diff
@@ -12,12 +12,14 @@
 import gov.healthit.chpl.dto.CertificationCriterionDTO;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
+import lombok.EqualsAndHashCode;
 
 @XmlType(namespace = "http://chpl.healthit.gov/listings")
 @XmlAccessorType(XmlAccessType.FIELD)
 @JsonIgnoreProperties(ignoreUnknown = true)
 @Builder
 @AllArgsConstructor
+@EqualsAndHashCode
```

Comment:
According to https://projectlombok.org/features/Builder this annotation can't exist on a class that has `@Builder` as well.

First paragraph in "Overview" after the list

Reply:
Updated with latest commit, thanks.